tests/xharness/Jenkins/Jenkins.cs

```diff
@@ -34,34 +35,7 @@ class Jenkins {
 		public bool Populating { get; private set; } = true;
 
 		public IHarness Harness { get; }
-		public bool IncludeAll;
```

Comment:
Something like this would have avoided a lot of changes...

```cs
        public bool IncludeAll { get { return TestSelection.IsEnabled (TestLabel.All); } }
```

Reply:
If you look at my initial commits, I went that path initially, and while it does remove the changes later, it has two thing I disliked:

1. A lot of code lines added not to change the code later.
2. Once we add more labels to individually select some tests we have to add more properties or have a mixed API with the properties and the method.